lib/utils/socketChainHelper.js

```diff
@@ -101,7 +101,7 @@ const getRequestType = (data, hasQuery = true) => {
 };
 
 const DEFAULT_COUNT = 1;
-const DEFAULT_DELAY = 1000;
+const DEFAULT_DELAY = 1;
```

Comment:
1ms ? Is it supported by BE?

Reply:
This weird 1s delay was introduced to account for slower TVs a few years ago.
This was a mistake. Changing this constant could have been in a separate PR however.